configuration/personattributetypes/personAttributeTypes.csv

```diff
@@ -6,22 +6,4 @@ c3bcd296-eb82-4451-a4b4-eedd1d5f322f,,impl.phoneNumber,impl.phoneNumber.desc,jav
 1df3e8c6-564e-4da9-9094-5d36f68100c9,,impl.distanceFromHealthCenter,impl.distanceFromHealthCenter.desc,org.openmrs.Concept,acf4c89a-1731-430a-aefd-a5db1fe96ee8,,,
 a4e9107d-175b-4e4d-b992-3ba155b94e4b,,impl.patientStatus,impl.patientStatus.desc,org.openmrs.Concept,cede5394-4d47-446f-85cc-e649673af6ea,,,
 f155a740-c252-4e45-bea6-51fd9706dcf7,,impl.other,impl.other.desc,java.lang.String,,,,
-82325788-3f10-11e4-adec-0800271c1b75,,givenNameLocal,Khmer_Given_Name,java.lang.String,,TRUE,,
-8233a58a-3f10-11e4-adec-0800271c1b75,,familyNameLocal,Khmer_Family_Name,java.lang.String,,TRUE,,
-8234e7b0-3f10-11e4-adec-0800271c1b75,,middleNameLocal,Khmer_Middle_Names,java.lang.String,,,,

```

Comment:
Maybe they keep showing in the UI because of the Bahmni Apps frontend config, but since they miss here submissions would fail?